lightningd/channel/channel.c

```diff
@@ -2002,6 +2032,7 @@ static void init_channel(struct peer *peer)
 				   &peer->short_channel_ids[LOCAL],
 				   &reconnected,
 				   &peer->unsent_shutdown_scriptpubkey,
+				   &peer->shutdown_sent[REMOTE],
```

Comment:
Doesn't seem to be a complete commit. I was looking for the place where `shutdown_send` was being set and this change seems not to be reflected in `peer_control.c`. Just noting this here so I can check later :wink: